openpype/hosts/houdini/api/pipeline.py

```diff
@@ -371,3 +428,35 @@ def main_take(no_update=True):
             instance_node.bypass(not new_value)
     except hou.PermissionError as exc:
         log.warning("%s - %s", instance_node.path(), exc)
+
+
+def list_instances(creator_id=None):
```

Comment:
It's used only in `HoudiniCreator` I would move it there (with `remove_instance`).